src/commands/synthetics/interfaces.ts

```diff
@@ -381,6 +381,7 @@ export interface TriggerConfig {
   config: UserConfigOverride
   id: string
   suite?: string
+  mobileApplicationVersion?: string
```

Comment:
What do you think about putting this one here? 
I was taking inspiration from our talks about `mobileApplicationVersionFilePath` and how it wasn't supposed to be actually part of the config but of the same level as `id`. 
However i'm not sure if it's better to have it here or in a separate class that this one merged with 🤔